### DISCUSSION TITLE
Dimension Groups and DimensionIn

### DISCUSSION THREAD

**Discussion by nsinisterra** [akki-io/laravel-google-analytics#23] [Ideas]
Hi there folks,

First of all, awesome work akki-io! Thanks for that!

I faced that actually we can't add "DimensionsIn" to GroupDimensions filters, actually GroupDimensions forces an [StringFilter](https://github.com/akki-io/laravel-google-analytics/blob/9eceb732f8bdb942a6b2446a755a868dd41a28ef/src/Traits/FilterByDimensionTrait.php#L108) filter for each array item. (I saw that this method is not exposed [in the facade ](https://github.com/akki-io/laravel-google-analytics/blob/9eceb732f8bdb942a6b2446a755a868dd41a28ef/src/Facades/LaravelGoogleAnalytics.php#L27)neither)

I was thinking that we may use a 4th parameter to the array, or maybe, we can accept an FilterExpression/Filter object to be added to the $filterExpressionList array directly.

What do you think? What's the best way to handle that? Are you accepting new PRs?

Regards,

**Comment by nsinisterra** [comment]
Ey @akki-io,

I suspect you are a little busy right now. 

Just to plan my release (deppends on yours), do you have any ETA for the new version?

Thanks!

**Comment by akki-io** [comment]
sorry @nsinisterra, I will try to get this on a release this week. It is merged to master so you can pull in the `dev-master` in your composer package.

**Reply by nsinisterra** [reply]
Ey @akki-io, just a fast check. Do you have any release date for the release?

Thanks!

**Reply by akki-io** [reply]
My apologies, I have been quite busy recently, but I will get this released this week. I appreciate your patience.

**Reply by akki-io** [reply]
https://github.com/akki-io/laravel-google-analytics/releases/tag/v1.6
